src/community/nmi.jl

```diff
@@ -0,0 +1,56 @@
+"normalised mutual information"
+function nmi(pa::Vector{Int}, pb::Vector{Int})
+    length(pa) == length(pb) || error("Two partitions have different number of nodes !")
+    n = length(pa)
+    qa = maximum(pa) # group number of partition a
+    qb = maximum(pb) # group number of partition b
+    (qa==1 && qb==1) && return 0.0
+    ga = zeros(Int, qa)
+    gb = zeros(Int, qb)
+    A = Array(Vector{Int}, qa)
+    B = Array(Vector{Int}, qa)
+    for i=1:qa
+        A[i] = Int[]
```

Comment:
The LightGraphs convention is to use `Vector{Int}()` instead of `Int[]`. Would you mind changing these constructors?